server/api/router.go

```diff
@@ -131,6 +131,11 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	clusterRouter.HandleFunc("/stores/limit", storesHandler.SetAllLimit).Methods("POST")
 	clusterRouter.HandleFunc("/stores/limit/scene", storesHandler.SetStoreLimitScene).Methods("POST")
 	clusterRouter.HandleFunc("/stores/limit/scene", storesHandler.GetStoreLimitScene).Methods("GET")
+	clusterRouter.HandleFunc("/stores/unsafe-recover",
```

Comment:
```suggestion
	clusterRouter.HandleFunc("/unsafe-recover",
```
let make it a first level command

Reply:
How about `/admin/unsafe-recover` or `/unsafe/recover`?

Reply:
@rleungx maybe `/usafe/remove-fail-stores`?